scripts/setup-action-configs.sh

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Required environment variables
+: "${CHANGE_TYPE:? "CHANGE_TYPE environment variable must be set"}"
+: "${MANAGEMENT_ACCOUNT:? "MANAGEMENT_ACCOUNT environment variable must be set"}"
+: "${BRANCH:? "BRANCH environment variable must be set"}"
+: "${INFRA_LIVE_REPO:? "INFRA_LIVE_REPO environment variable must be set"}"
+: "${WORKING_DIRECTORY:? "WORKING_DIRECTORY environment variable must be set"}"
+: "${COMMAND:? "COMMAND environment variable must be set"}"
+: "${ARGS:? "ARGS environment variable must be set"}"
+
+# Optional environment variables
+NEW_ACCOUNT_NAME="${NEW_ACCOUNT_NAME:-}"
+TEAM_ACCOUNT_NAMES="${TEAM_ACCOUNT_NAMES:-}"
+CHILD_ACCOUNT_ID="${CHILD_ACCOUNT_ID:-}"
+PIPELINES_AUTH_ROLE="${PIPELINES_AUTH_ROLE:-}"
+
+check_pipeline_is_installed() {
+    if ! command -v pipelines &> /dev/null; then
+        echo "pipelines CLI not installed"
+        exit 1
+    fi
+}
+
+presign_caller_identity_token() {
+    check_pipeline_is_installed
+
+    pipelines auth presign
+}
+
+handle_account_request() {
+    readonly management_account="$1"
+    readonly new_account_name="$2"
+    readonly branch="$3"
+    readonly infra_live_repo="$4"
+    readonly working_directory="$5"
+    readonly terragrunt_command="$6"
+    readonly pipelines_auth_role="$7"
+
+    echo "workflow=create-account-and-generate-baselines.yml" >> "$GITHUB_OUTPUT"
+    workflow_inputs="$(jq -c -n \
+        --arg management_account "$management_account" \
+        --arg new_account_name "$new_account_name" \
+        --arg branch "$branch" \
+        --arg infra_live_repo "$infra_live_repo" \
+        --arg working_directory "$working_directory" \
+        --arg terragrunt_command "$terragrunt_command" \
+        '{
+            "management_account": $management_account,
+            "new_account_name": $new_account_name,
+            "branch": $branch,
+            "infra_live_repo": $infra_live_repo,
+            "working_directory": $working_directory,
+            "terragrunt_command": $terragrunt_command
+        }'
+    )"
+    if [[ -n "$pipelines_auth_role" ]]; then
+        presigned_caller_identity="$(presign_caller_identity_token)"
+        workflow_inputs="$(jq --arg presigned_caller_identity "$presigned_caller_identity" '. + {presigned_caller_identity: $presigned_caller_identity}' <<< "$workflow_inputs")"
```

Comment:
NIT: Seems like these two lines are repeated a few times. Extract to a function?